components/esp_lvgl_port/test_apps/simd/main/test_lv_fill_functionality.c

```diff
@@ -147,10 +155,38 @@ TEST_CASE("Test fill functionality RGB565", "[fill][functionality][RGB565]")
     functionality_test_matrix(&test_matrix, &test_case);
 }
 
+TEST_CASE("Test fill functionality RGB888", "[fill][functionality][RGB888]")
+{
+    test_matrix_params_t test_matrix = {
+        .min_w = 8,             // 8 is the lower limit for the esp32s3 asm implementation, otherwise esp32 is executed
+        .min_h = 1,
+        .max_w = 16,
+        .max_h = 16,
+        .min_unalign_byte = 0,
+        .max_unalign_byte = 16,
+        .unalign_step = 1,
+        .dest_stride_step = 1,
+        .test_combinations_count = 0,
+    };
+
+    func_test_case_params_t test_case = {
+        .blend_api_px_func = &lv_draw_sw_blend_color_to_rgb888,
+        .color_format = LV_COLOR_FORMAT_RGB888,
+        .data_type_size = 3,
```

Comment:
I would prefer sizeof here
```suggestion
        .data_type_size = sizeof(uint8_t) * 3,   // 24-bit data length
```